packages/ui/Scrollable/Scrollable.js

```diff
@@ -204,6 +204,30 @@ class ScrollableBase extends Component {
 		 */
 		noScrollByWheel: PropTypes.bool,
 
+		/**
+		 * Called when trigerring a drag event.
+		 *
+		 * @type {Function}
+		 * @private
```

Comment:
I was going to remove the `@private` since it's redundant and, if we opened this, would want it public, I think... but...  I'd have to update all the other props, too.  So, I didn't do it now.